src/ui/playmode-sub-panel.cpp

```diff
@@ -353,8 +356,9 @@ void PlayModeSubPanel::showPortaContinuationMenu()
     p.addSectionHeader("Portamento Continuation");
     p.addSectionHeader("Experimental; May change before 1.1");
```

Comment:
Does this mean we can remove this 'experimental' warning also? 

And thank you!